tests/ca-less/certificates/dirsrv/ipaserver/cert.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We should not add generated certificates anymore. With your script the certificates could be generated easily. Additionally we are not using the same domains for the tests all the time, therefore the domain should not be pinned down.